base/applications/rapps/gui.cpp

```diff
@@ -557,6 +557,11 @@ CMainWindow::OnCommand(WPARAM wParam, LPARAM lParam)
             case ID_CHECK_ALL:
                 m_ApplicationView->CheckAll();
                 break;
+
+            case ID_ACTIVATE_APPWIZ:
```

Comment:
Why is this needed even with the initial `SelectItem` call to the correct tree node in the `InitCategoriesList` function above?